src/index.html

```diff
@@ -34,6 +34,42 @@ <h2 class="navbar__subtitle">a travel journal</h2>
       </div>
     </main>
 
+    <form id="add-city-form" class="container">
+      <p id="cityName">City</p>
+      <input type="text" onfocus="this.value=''" name="text" id="inputCity" />
+      <p id="countryName">Country</p>
+      <input
+        type="text"
+        onfocus="this.value=''"
+        name="text"
+        id="inputCountry"
+      />
+      <!-- add photo field-->
+      <input type="file" multiple="false" accept="image/*" id="inputImg" />
+      <!-- radio buttons-->
+      <div class="radio-buttons">
+        <input
+          id="rd-visited"
+          type="radio"
+          name="btn"
+          value="Visited"
+          label="Visited"
+        />
+        <label for="Visited">Visited</label>
+        <input
+          id="rd-bucketlist"
+          type="radio"
+          name="btn"
+          value="Bucketlist"
+          label="Bucketlist"
+        />
+        <label for="Bucketlist">Bucketlist</label>
+      </div>
+      <!-- Upload button -->
+      <button type="submit" class="upload">Upload</button>
+    </form>
+
+    <script src="scripts/destinationList.js"></script>
```

Comment:
It's not need to import `destinationList` here because it's already  imported at `main.js`